src/Compilers/Core/Portable/Symbols/ITypeComparable.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Internal interface that allows a symbol to declare that it supports comparisons involving type symbols
```

Comment:
why internal?  is this not something external consumers would want to use?

Reply:
If you can't implement a symbol externally how would you use this interface?